tests/wpt/web-platform-tests/fetch/http-cache/vary.html

```diff
@@ -245,11 +246,12 @@
         ]
       },
       {
-        name: "HTTP cache doesn't use three-way Vary response when request omits variant header.",
+        name: "HTTP cache doesn't use three-way Vary response when request doesn't match, regardless of header order.",
```

Comment:
Are we losing coverage for the original test by changing it like this?

Reply:
I don't think so, I wrote the test when I realized my code was relying on the order of headers, and it failed while the other would still pass.

Reply:
by the way, I left the original test in there.